synapse/storage/roommember.py

```diff
@@ -57,6 +59,49 @@
 
 
 class RoomMemberWorkerStore(EventsWorkerStore):
+    def __init__(self, db_conn, hs):
+        super(RoomMemberWorkerStore, self).__init__(db_conn, hs)
+
+        # Is the current_state_events.membership up to date? Or is the
+        # background update still running?
+        self._current_state_events_membership_up_to_date = False
+
+        txn = LoggingTransaction(
+            db_conn.cursor(),
+            name="_check_safe_current_state_events_membership_updated",
+            database_engine=self.database_engine,
+            after_callbacks=[],
```

Comment:
I kinda feel like `None` would be better here, then if someone tries to do a `.call_after` from a txn function, it'll blow up rather than secretly not do the invalidation actions.

... and then I kinda feel like None should be the default for those params on LoggingTransaction, so the boilerplate is reduced.